### ISSUE TITLE
Vcpkg android prefab not work

### ISSUE THREAD

**Issue by rogerou** [microsoft/vcpkg#25720]
**Describe the bug**
vcpkg export prefab will ask export vcpkg-cmake:x64-windows at the same time
**Environment**
- OS:Windows
- Compiler: revision NDK 23b

**To Reproduce**
1. ./vcpkg install jsoncpp:arm64 ....l
2. ./vcpkg export jsoncpp --triplet arm64-android --prefab --prefab
The following packages are already built and will be exported:
    jsoncpp:arm64-android
  * vcpkg-cmake:x64-windows
  * vcpkg-cmake-config:x64-windows
Additional packages (*) need to be exported to complete this operation.

Exporting package vcpkg-cmake...
[DEBUG]
        Writing manifest
        To C:\Users\ouzhijie\Documents\vcpkg\prefab\vcpkg-cmake\aar\AndroidManifest.xml
        Writing prefab meta data
        To C:\Users\ouzhijie\Documents\vcpkg\prefab\vcpkg-cmake\aar\prefab\prefab.json

[DEBUG] Found 4 triplets
        arm-android
        arm64-android
        x64-android
        x86-android

Error: Packages not installed vcpkg-cmake:arm-android C:\Users\ouzhijie\Documents\vcpkg\installed\vcpkg\info\vcpkg-cmake_2022-07-02_arm-android.list
Repro code when

**Expected behavior**

**Failure logs**
-(please attached failure logs)

**Additional context**

**Comment by JackBoosY** [comment]
You should use Android triplet I think?

**Comment by glanium** [comment]
what is vcpkg-cmake????
This package cause all failers...
This package is required to export ??????

**Comment by glanium** [comment]
When install,vcpkg requires non-android triplet vcpkg-cmake and vcpkg-cmake-config
But when exporting to prefab,it requires android-triplet vcpkg-cmake....

Exporting to prefab completey does not work??

**Comment by JackBoosY** [comment]
Maybe you need to pass extra option `--host-triplet=arm64-windows` in the install command if your environment is arm64.

**Comment by JackBoosY** [comment]
We hope your question was answered to your satisfaction; if it wasn't, you can reopen with more info.

**Comment by cltien** [comment]
I got the exact errors, the message I got is like this:

--------------------------------------------------------
vcpkg export --triplet x64-android jsoncpp --prefab
The following packages are already built and will be exported:
    jsoncpp:x64-android
  * vcpkg-cmake:x64-windows
  * vcpkg-cmake-config:x64-windows
Additional packages (*) need to be exported to complete this operation.
Exporting vcpkg-cmake...
error: Your vcpkg 'installed' tree is corrupted.
--------------------------------------------------------

Can this be reopened since the error still exists?
